starters/blog/gatsby-config.js

```diff
@@ -53,7 +60,57 @@ module.exports = {
         //trackingId: `ADD YOUR TRACKING ID HERE`,
       },
     },
-    `gatsby-plugin-feed`,
+    {
+      resolve: `gatsby-plugin-feed`,
+      options: {
+        query: `
```

Comment:
should it make sense to make a new `gatsby-plugin-feed-mdx` for this use case
(maybe use the `gatsby-plugin-feed` but overwrite somehow the default query)

Reply:
i think it can be useful for other mdx sites

Reply:
@muescha good idea!

Reply:
@muescha I made `gatsby-plugin-feed-mdx` – https://github.com/thomaswangio/gatsby-plugin-feed-mdx. will replace it here as well as in the blog post!

Reply:
tested it here https://github.com/thomaswangio/gatsby-personal-starter-blog/commit/4daab2d1609c560b62a9580e9d4fe16b878c4fe8 and it works!

Reply:
New PR: https://github.com/gatsbyjs/gatsby/pull/19580

Reply:
A sidenote: You don't need to close a PR.  Just commit new or changed files to the same branch - then the changes will show up here. 

-> That is more convenient to review. Reviewer only need to check the new commits and not the hole PR

Reply:
I see you make a new PR because you removed the fork to early